R/utils.R

```diff
@@ -36,6 +36,28 @@ remove_enclosing_curly_braces <- function(x) {
   }
 }
 
+#' Suppresses plot display in the IDE by opening a PDF graphics device
+#'
+#' This function opens a PDF graphics device using \code{\link[grDevices]{pdf}} to suppress
+#' the plot display in the IDE. The purpose of this function is to avoid opening graphic devices
+#' directly in the IDE.
+#'
+#' @param reactive_expr A reactive expression or any other R code that generates the plot(s) on IDE.
```

Comment:
I think definition is much deeper and doesn't limit just to reactive expression. This function is a wrapper on force which avoind plot rendering. That's why I'd just use `x` argument 
```suggestion
#' @param x lazy binding which generates the plot(s)
```